lib/Rex/Virtualization/LibVirt/create.pm

```diff
@@ -83,7 +83,7 @@ sub execute {
       my $size = $_->{'size'};
       if ( !is_file( $_->{"file"} ) ) {
         Rex::Logger::debug("creating storage disk: \"$_->{file}\"");
-        i_run "$QEMU_IMG create -f raw $_->{'file'} $size";
+        i_run "$QEMU_IMG create -f $_->{driver_type} $_->{'file'} $size";
```

Comment:
Can we add a note about this to the documentation, please? Something like `With LibVirt, the same driver types are supported as formats qemu-img supports.`

ps.: looks like `driver_type` is not included at all in the current examples in `Rex::Commands::Virtualization`